lib/hanami/assets/helpers.rb

```diff
@@ -186,9 +240,25 @@ def javascript(*sources)
       #   <%= stylesheet 'application' %>
       #
       #   # <link href="https://assets.bookshelf.org/assets/application-28a6b886de2372ee3922fcaf3f78f2d8.css" type="text/css" rel="stylesheet">
-      def stylesheet(*sources)
+      def stylesheet(*sources, **options)
         _safe_tags(*sources) do |source|
-          html.link(href: _typed_asset_path(source, STYLESHEET_EXT), type: STYLESHEET_MIME_TYPE, rel: STYLESHEET_REL).to_s
+          tag_options = options.merge({
```

Comment:
@jodosha I tried to remain as close as possible to the original implementation, and this was true then. If a developer wanted to be overwrite the `href`, `type`, and `rel` attributes, they could simply use the HTML helper. However, it could be possible to extend this to allow that sort of fine-grained control, although there would be the need to guard against specific misuse. For example, it wouldn't make sense for `href` to be specified when multiple sources were provided, for example.